fdbserver/include/fdbserver/DataDistribution.actor.h

```diff
@@ -250,6 +251,20 @@ struct GetTopKMetricsRequest {
 	                      double minBytesReadPerKSecond = 0)
 	  : topK(topK), keys(keys), maxBytesReadPerKSecond(maxBytesReadPerKSecond),
 	    minBytesReadPerKSecond(minBytesReadPerKSecond) {}
+
+	// Return true if a.score > b.score, return the largest topK in keys
+	static bool compare(const GetTopKMetricsReply::KeyRangeStorageMetrics& a,
```

Comment:
It looks like this function is not used anywhere (nor is the old `comparator` function). Is there a plan to use this in the future?